src/schemas/types.js

```diff
@@ -1,20 +1,97 @@
+export const typeFormatChoices = [
+  {
+    'key': 'int32',
+    'label': '32-bit integer',
+    'conditions': {
+      '../type': 'integer'
+    }
+  },
+  {
+    'key': 'int64',
+    'label': '64-bit integer',
+    'conditions': {
+      '../type': 'integer'
+    }
+  },
+  {
+    'key': 'float',
+    'label': '32-bit floating point',
+    'conditions': {
+      '../type': 'number'
+    }
+  },
+  {
+    'key': 'double',
+    'label': '64-bit floating point',
+    'conditions': {
+      '../type': 'number'
+    }
+  },
+  {
+    'key': '',
```

Comment:
As specified in http://swagger.io/specification/#data-types-12, normal strings have no format.